test/cache_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class TestOpenExchangeRates < Test::Unit::TestCase
+  def test_cache_key
+    cache = OpenExchangeRates::Cache::MemoryCache.new
+    assert_equal 'open-exchange-rates:2015-12-27:USD', cache.key(:from => 'USD', :on => '2015-12-27')
+  end
+
+  def test_cache_set_returns_value_for_null_adapter
+    cache = OpenExchangeRates::Cache::MemoryCache.new
+    assert_equal 'value', cache.set('key', 'value')
+    assert_equal 'value', cache.get('key')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.